meteor/__mocks__/helpers/database.ts

```diff
@@ -374,27 +381,33 @@ export interface DefaultEnvironment {
 
 	ingestDevice: PeripheralDevice
 }
-export function setupDefaultStudioEnvironment(): DefaultEnvironment {
+export function setupDefaultStudioEnvironment(organizationId?: OrganizationId | null): DefaultEnvironment {
```

Comment:
I think you can write these two lines as:
```
export function setupDefaultStudioEnvironment(organizationId: OrganizationId | null = null): DefaultEnvironment {
```
and it will have the same effect, and make the default more obvious